src/java.base/share/classes/sun/net/NetProperties.java

```diff
@@ -68,10 +68,11 @@ private static void loadDefaultProperties() {
             File f = new File(fname, "conf");
             f = new File(f, "net.properties");
             fname = f.getCanonicalPath();
-            InputStream in = new FileInputStream(fname);
-            BufferedInputStream bin = new BufferedInputStream(in);
-            props.load(bin);
-            bin.close();
+            try (FileInputStream in = new FileInputStream(fname);
+                 BufferedInputStream bin = new BufferedInputStream(in))
+            {
```

Comment:
Style-wide I'd probably put the "{" at the end of L72.

Reply:
As you wish.
But I personally like to have `{` on separate line, in case of multi-line `try`. It makes clear when `try` header is actually finished and body starts.